wire-ios/Tests/Mocks/MockConversation.swift

```diff
@@ -74,6 +74,8 @@ class SwiftMockConversation: NSObject, Conversation {
     var areServicesPresent: Bool = false
 
     var domain: String?
+
+    var isConversationEligibleForVideoCalls = false
```

Comment:
Do we use this property? Looks like it's not related to this PR

Reply:
Good catch. Removed it